Content.Server/Atmos/EntitySystems/AtmosphereSystem.Hotspot.cs

```diff
@@ -137,12 +137,16 @@ private void HotspotExpose(GridAtmosphereComponent gridAtmosphere, TileAtmospher
 
             var plasma = tile.Air.GetMoles(Gas.Plasma);
             var tritium = tile.Air.GetMoles(Gas.Tritium);
+            // AruMoon edit start
+            var hydrogen = tile.Air.GetMoles(Gas.Hydrogen);
+            var hypernoblium = tile.Air.GetMoles(Gas.HyperNoblium);
+            // AruMoon edit end
 
             if (tile.Hotspot.Valid)
             {
                 if (soh)
                 {
-                    if (plasma > 0.5f || tritium > 0.5f)
+                    if (plasma > 0.5f && hypernoblium < 5f || tritium > 0.5f && hypernoblium < 5f || hydrogen > 0.5f && hypernoblium < 5f) // AruMoon edit
```

Comment:
```suggestion
                    if ((plasma > 0.5f || tritium > 0.5f || hydrogen > 0.5f) && hypernoblium < 5f) // AruMoon edit
```
Исправляем для будущей читабельности